apps/meteor/app/models/server/models/Subscriptions.js

```diff
@@ -12,8 +12,8 @@ export class Subscriptions extends Base {
 	constructor(...args) {
 		super(...args);
 
-		this.tryEnsureIndex({ rid: 1 });
 		this.tryEnsureIndex({ rid: 1, ls: 1 });
+		this.tryEnsureIndex({ 'userHighlights.0': 1, 'rid': 1 });
```

Comment:
what if you revert the changes to indexes and we do them in another PR?

Reply:
Fair :eyes: